wolfcrypt/src/aes.c

```diff
@@ -8850,7 +8842,7 @@ static WARN_UNUSED_RESULT int wc_AesGcmDecrypt_STM32(
         authPadSz = authInSz;
     }
 
-#ifdef WOLFSSL_STM32MP13
+#if defined(WOLFSSL_STM32MP13)
```

Comment:
Given the previous changes, I'm assuming the H7S HAL has the same minimum length bug in their GCM code, in which case shouldn't this also be `STM_CRYPT_HEADER_TYPE`?

Reply:
The only requirement for H7S is add auth must be multiple of 4 bytes. Its fine if it's less than 16.